bamMergePreprocessing.wdl

```diff
@@ -241,40 +237,32 @@ task preprocessBam {
   input {
     Boolean doFilter = true
     Boolean doMarkDuplicates = true
-    Boolean doSplitNCigarReads = false
-
     String outputFileName
 
     # by default write tmp files to the current working directory (cromwell task directory)
     # $TMPDIR is set by Cromwell
     # $TMP is set by Univa
     String temporaryWorkingDir = ""
 
-    Array[File] bams
-    Array[File] bamIndexes
+    File inputBam
+    File inputBamIndex
     Array[String] intervals
 
     # filter parameters
     String filterSuffix = ".filter"
     Int filterFlags = 260
     Int? minMapQuality
-    String? filterAdditionalParams
+    String? filterAdditionalParams 
 
-    # mark duplicates
-    String markDuplicatesSuffix = ".deduped"
+    # markDuplicates parameters
+    String dedupSuffix = ".deduped"
     Boolean removeDuplicates = false
     Int opticalDuplicatePixelDistance = 100
     String? markDuplicatesAdditionalParams
 
-    # split N cigar reads
-    String splitNCigarReadsSuffix = ".split"
     String reference
-    Boolean refactorCigarString = false
-    Array[String] readFilters = []
-    String? splitNCigarReadsAdditionalParams
-
-    Int jobMemory = 24
-    Int overhead = 6
+    Int jobMemory = 48
+    Int overhead = 8
     Int cores = 1
     Int timeout = 6
     String modules = "samtools/1.9 gatk/4.1.6.0 python/2.7"
```

Comment:
Is it possible to update to `python/3.10.6`? Python 2.7 is EOL.

Reply:
Just found python oly used collectFilesBySample and indelRealign task, which have been removed from wdl, now we can remove python altogether.